libs/server/Objects/ItemBroker/CollectionItemResult.cs

```diff
@@ -20,6 +20,12 @@ public CollectionItemResult(byte[] key, byte[][] items)
             Items = items;
         }
 
+        public CollectionItemResult(byte[] key, (double Score, byte[] Element)[] scoredItems)
```

Comment:
I would like to create a separate PR in future to remove `CollectionItemResult` class and replace it with `SpanByteAndMemory`, Let me know if there is any issue with that